omnigibson/utils/transform_utils_np.py

```diff
@@ -0,0 +1,1345 @@
+"""
+Utility functions of matrix and vector transformations.
+
+NOTE: convention for quaternions is (x, y, z, w)
+"""
+
+import math
+
+import numpy as np
+from numba import jit, prange
+from scipy.spatial.transform import Rotation as R
+
+PI = np.pi
+EPS = np.finfo(float).eps * 4.0
+
+# axis sequences for Euler angles
+_NEXT_AXIS = [1, 2, 0, 1]
+
+# map axes strings to/from tuples of inner axis, parity, repetition, frame
+_AXES2TUPLE = {
```

Comment:
make sure one-to-one correspondence with `transform_utils.py`

Reply:
done